modules/Rock.cmake

```diff
@@ -476,7 +476,8 @@ function(rock_install_headers HEADER_LIST)
     # otherwise the function would need to be called with a quoted list, e.g.
     # rock_install_headers("${MY_LIST}")
     foreach(HEADER ${ARGV})
-        string(REGEX MATCH "(.*)[/\\]" DIR ${HEADER})
+        string(REPLACE "${CMAKE_CURRENT_BINARY_DIR}/" "" HEADER_relative "${HEADER}")
```

Comment:
See updated PR description ... hope that will be enough information this time ;-)